README.md

```diff
@@ -173,6 +192,14 @@ Possible values of this variable:
 * `{state: absent}`: the ansible host name is unset in the insights-client config file and Host Based Inventory (HBI) is updated to use the system host name.
 * any other string value: the ansible host name is changed in Host Based Inventory (HBI).
 
+```yaml
+    rhc_inights:
+      baseurl: "cert-api.access.redhat.com:443/r/insights"
+```
+
+Configures the Base URL for the Insights API.
+Defaults to "cert-api.access.redhat.com:443/r/insights".
```

Comment:
@ptoscano Is this value built-in to the insights client?  where does it come from?

Reply:
It's the default value from `/etc/insights-client/insights-client.conf`:

```
# Base URL for the Insights API
#base_url=cert-api.access.redhat.com:443/r/insights
```